app/models/customer.rb

```diff
@@ -2,15 +2,15 @@ class Customer < ActiveRecord::Base
   acts_as_taggable
 
   belongs_to :enterprise
-  belongs_to :user, class_name: Spree.user_class
-  has_many :orders, class_name: Spree::Order
+  belongs_to :user, class_name: Spree.user_class.to_s
```

Comment:
I vaguely remembered issues where the test suite still used an alternate class in some places, but I think it might have been removed now. `Spree::LegacyUser`?

Reply:
Interesting, we still have a bit of code referencing it, but the class doesn't exist any more... :see_no_evil:

Reply:
I think current_api_user in ApiHelper is dead code. Deleted it here, will watch the build:
https://github.com/openfoodfoundation/openfoodnetwork/pull/7066/files